src/mappings/multiTokens/events/collection_created.ts

```diff
@@ -72,9 +77,39 @@ async function getCallData(ctx: CommonContext, call: Call) {
                 explicitRoyaltyCurrencies,
             }
         }
+        throw new UnknownVersionError(data.constructor.name)
+    } else if (call.name === 'FuelTanks.dispatch_and_touch' || call.name === 'FuelTanks.dispatch') {
+        let data: FuelTanksDispatchCall | FuelTanksDispatchAndTouchCall
+        if (call.name === 'FuelTanks.dispatch') {
+            data = new FuelTanksDispatchCall(ctx, call)
+        } else {
+            data = new FuelTanksDispatchAndTouchCall(ctx, call)
+        }
+
+        if (
+            data.isMatrixEnjinV603 &&
+            data.asMatrixEnjinV603.call.__kind === 'MultiTokens' &&
+            data.asMatrixEnjinV603.call.value.__kind === 'create_collection'
+        ) {
+            const { descriptor } = data.asMatrixEnjinV603.call.value
+            const { maxTokenCount, maxTokenSupply, forceSingleMint } = descriptor.policy.mint
+            const royalty = descriptor.policy.market?.royalty
+            const market = royalty ? await getMarket(ctx, royalty) : null
+            const { explicitRoyaltyCurrencies } = descriptor
+
+            return {
+                maxTokenCount,
+                maxTokenSupply,
+                forceSingleMint,
+                market,
+                explicitRoyaltyCurrencies,
+            }
+        }
 
         throw new UnknownVersionError(data.constructor.name)
     }
+
+    throw new Error(`${call.name} is not supported`)
```

Comment:
Can we make a custom error in the same class as the UnknownVersion to handle this in a standard way in another classes?